### PR TITLE
Installer guide and play test feedback

### PR DESCRIPTION
**Pull request by gdamron**
- Adds a short section describing how to use the installer (please make sure the link works!)
- Incorporates feedback from the last play test

One request was to create an overview page for Enklu Embedded. That's coming soon, but I wanted to make the installer available as a soon as possible.

### CODE REVIEW

**Review by jsundra — APPROVED** [review]
👍

**Review by ShannonKalei — APPROVED** [review]

File: docs/Embedded/EmbeddedSetup.md [hunk 1]
```diff
@@ -7,9 +7,27 @@ title: Installing Enklu Embedded
...
+
+To make getting started as seamless as possible, we have created an installer that automatically imports the Enklu Embedded package and all its dependencies:
+
+[`EnkluEmbeddedInstaller.unitypackage`](https://cdn.enklu.com/web-assets/packages/EnkluEmbeddedInstaller.unitypackage)
+
+To run the installer, simply drag it into your Unity project or import it from the `Asset > Import Package > Custom Package...` menu and confirm the two files to import.
+
+![Importing the Installer](/img/product/embedded/Installer.png)
+
+Once the files have been added to the project, the installer will import the Enklu Embedded package, if necessary. Please note that the process can take several minutes. It will perform this check every time you launch your Unity project. It is also possible to run the `Enklu > Install Enklu Embedded...` menu command.
```
Comment (liam-philiben) [inline docs/Embedded/EmbeddedSetup.md:24]: Not sure if you need the "if necessary" here, could probably do without it

Comment (liam-philiben) [inline docs/Embedded/EmbeddedSetup.md:24]: Also, maybe this is more of my ignorance on the subject, but what exactly is performing the check every time I launch my Unity project? Is it the package manager? Or is it the installer?

File: docs/Embedded/EnkluVolume.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-title: Embedding an Experience
+title: Embedding an Experience in Your Unity Scene
 ---
 
 To embed an Enklu Cloud experience in your Unity project, add an EnkluVolume prefab to your scene. Find it by searching for "EnkluVolume" in the project tab and ensuring the "In Packages" option is selected. Once the prefab is in your scene, select it and log in to your Enklu Cloud account in the inspector tab.
 
 ![Enklu Volume](/img/product/embedded/VolumeImport.gif)
 
-After you've logged in, use the inspector tab to set an existing experience to embed or create a new one. You can embed an entire Enklu experience in your scene or just part of one by selecting or creating an element.
+After you've logged in, use the inspector tab to set up your Enklu Volume. You can embed an entire experience or just part of one. Select an element from the dropdown menus to choose which part.
```
Comment (liam-philiben) [inline docs/Embedded/EnkluVolume.md:10]: I think it's just a singular dropdown menu, right? So maybe this sentence should be "Select an element from the dropdown menu to choose which part of your experience you would like to embed" or something to that effect

File: docs/Embedded/EnkluVolume.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
 
-> Every Enklu Experience has an element named "root." This is the first entry in the scene's hierarchy, and all other elements are its children.
+> Every Enklu Experience has an element named "root." If you select the "root" element, the entire experience will be included in your Volume.
 
-After selecting an experience and an element, enter Play Mode by clicking the Play button on the Toolbar. The EnkluVolume will fetch the latest version of the experience and begin receiving real-time updates. The content will load alongside anything already existing in the scene.
+After selecting an experience and an element from the inspector, enter Play Mode by clicking the Play button on the Toolbar. The EnkluVolume will fetch the latest version of the Enklu experience and begin receiving real-time updates. The content will load alongside anything already existing in the Unity scene.
```
Comment (liam-philiben) [inline docs/Embedded/EnkluVolume.md:14]: Super minor comment, but I think "...will load alongside anything that already exists" reads a little bit better than "anything already existing"

**Review by liam-philiben — CHANGES_REQUESTED** [review]
Overall, looks good! Like last time, made some super nitpicky and pedantic comments re: sentences and word usage, but content-wise everything looks really solid!

File: docs/Embedded/EmbeddedSetup.md (hunk 1 above)
Comment (gdamron) [reply docs/Embedded/EmbeddedSetup.md:24]: will clarify

File: docs/Embedded/EnkluVolume.md (hunk 2 above)
Comment (gdamron) [reply docs/Embedded/EnkluVolume.md:10]: It _is_ multiple dropdowns, but using "menus" plural does seem like a typo.

**Review by liam-philiben — APPROVED** [review]

**Pull request merged**